### PR TITLE
Issue 31 - fix nova conf dir creation

### PR DESCRIPTION
**Pull request by barryoneill**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -10,8 +11,9 @@
 for name in glob.glob('config/plugins.d/*.conf'):
     conf.insert(1, name)
 
-if not os.path.exists('~/.nova/'):
-    os.makedirs('~/.nova/')
+nova_confdir = expanduser("~/.nova")
+if not os.path.exists(nova_confdir):
+    os.makedirs(nova_confdir)
```
Comment (gheine) [inline setup.py]: Also needs to be changed on line 102

Comment (jkenny) [reply setup.py]: Does Nova require this to exist? The [docs](http://gilt-nova.readthedocs.io/en/latest/getting_started.html) suggest it's optional.

Comment (barryoneill) [reply setup.py]: Well, considering this piece of code simply doesn't work (on mac anyway) I'd argue it's optional alright :)

**Comment by barryoneill** [comment]
This will mean that new users were also missing the default nova.conf, and any users who've installed the 'broken' versions, will be missing this file even after this change (I think)..

**Pull request merged**